docs/build_custom_logstash_image.md

```diff
@@ -0,0 +1,62 @@
+# Build A Custom Image with Logstash
+
+If you are looking for a logstash image version with opensearch plugin not available in the [official Docker repository tags](https://hub.docker.com/r/opensearchproject/logstash-oss-with-opensearch-output-plugin/tags) or if you want to include other plugins in your image, it is possible to build a custom image.
+
+## Logstash 8.x
+
+Below is an example of a `Dockerfile` to install `Opensearch 2.0.2` for **`Logstash 8.x`**
+
+``` Dockerfile
+ARG APP_VERSION
+
+FROM docker.elastic.co/logstash/logstash-oss:${APP_VERSION}
+RUN logstash-plugin install --version 7.1.1 logstash-integration-aws
+RUN logstash-plugin install --version 2.0.1 logstash-output-opensearch
+```
+
+## Logstash 7.x
+
+For **`Logstash 7.x`** , the `logstash output opensearch gem` needs to be build.
+
+### Building logstash output opensearch gem plugin
+
+1. Clone `logstash-output-opensearch repo` for the plugin version we want to build. For exemple , clone the  [v2.0.2](https://github.com/opensearch-project/logstash-output-opensearch/tree/2.0.2) 
+
+2. Remove [this line that adds the json version spec](https://github.com/opensearch-project/logstash-output-opensearch/blob/2.0.2/logstash-output-opensearch.gemspec#L49) 
+
+3. Build the gem by running the following command:
+
+    ```bash
```

Comment:
Align left, add a space after $.

Reply:
Thank you. I will standardize it to match the format used in other documentation files within the repository.

Reply:
Done